bazel/grpc_python_deps.bzl

```diff
@@ -41,13 +40,12 @@ def grpc_python_deps():
         )
 
     if "io_bazel_rules_python" not in native.existing_rules():
-        git_repository(
-            name = "io_bazel_rules_python",
-            commit = "fdbb17a4118a1728d19e638a5291b4c4266ea5b8",
-            remote = "https://github.com/bazelbuild/rules_python.git",
+        http_archive(
+            name = "rules_python",
```

Comment:
You need to keep the old name if you aren't going to rename the usages.

Reply:
Good catch, renamed.